tasks.py

```diff
@@ -7,6 +7,12 @@
 
 ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__)))
 
+LANGUAGES = ['fr', 'pt']
```

Comment:
Why only those?

Reply:
Because they currently are the only translated languages on `udata-ods`.
We can add toher translations later on demand